.gitignore

```diff
@@ -1,3 +1,5 @@
+data/.DS_Store
+.DS_Store
```

Comment:
Let's replace these two lines with: 
**/.DS_Store
to exclude .DS_Store files from all directories

Reply:
Updated.